lib/wallets/crypto_currency/coins/wownero.dart

```diff
@@ -92,7 +92,7 @@ class Wownero extends CryptonoteCurrency {
   bool get hasMnemonicPassphraseSupport => false;
 
   @override
-  List<int> get possibleMnemonicLengths => [defaultSeedPhraseLength, 25];
+  List<int> get possibleMnemonicLengths => [defaultSeedPhraseLength, 16, 25];
```

Comment:
Wownero uses a seed length of 14.  Edit: correction, 14, 16, and 25 are all valid: https://guides.cakewallet.com/docs/cryptos/wownero/#seed-format

Reply:
Wowlet generates seeds of length 14, Cake generates 16 by default.